ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -1,10 +1,14 @@
 import React, { ReactElement } from 'react';
 import { useHistory } from 'react-router-dom';
 import {
+    Alert,
+    Badge,
     Button,
     EmptyState,
     EmptyStateIcon,
     EmptyStateVariant,
+    ExpandableSection,
```

Comment:
For my understanding, do you know pro and con of alternatives which seem to overlap?

It looks like you made a practical choice:
* https://www.patternfly.org/v4/components/form/design-guidelines#field-groups
    > Field groups sort form fields into hierarchical, collapsible groups based on theme. All field groups are nested and indented by default, so they create a clearer information architecture between groups. Since all elements won't be visible at all times, always title field groups based on their purpose.
    See below how you do **always title field groups**.
    If every group is already separated by border, then **indented by default** seems unneeded.
* https://www.patternfly.org/v4/components/expandable-section/design-guidelines
    > An expandable section component is used to support progressive disclosure in a form or page by hiding additional content when you don't want it to be shown by default. An expandable section can contain any type of content such as plain text, form inputs, and charts.
    Displaying the section heading while collapsed, instead of **Show whatever** in examples, removes my main potential concern with more generic `ExpandableSection` element.

Reply:
A subtle visual improvement might be if collapse/expand icon aligned with heading line instead of vertical centered alignment with heading and descriptive sentence.